README.md

```diff
@@ -54,3 +54,80 @@ spark.optimizeSpatialAuto(sourceTable, outputTable, outputLocation)
 // optimize with the user defined block size
 spark.optimizeSpatial(sourceTable, outputTable, outputLocation, blockSize = 20097000)
 ```
+
+
+### Enabling Carto Query Optimizations on Databricks
+
+> This section and approach are based on [docs](https://sedona.apache.org/setup/databricks/)
+> from Apache Sedona.
+
+
+#### Create Initialization Script
+
+First, write a script to DBFS which can be used to copy jars from
+[DBFS](https://docs.databricks.com/data/databricks-file-system.html) to the local filesystem
+on master.
```

Comment:
I think that is not on _master_, that's the cluster wide init script; lets say smth like "to [the default class path](https://kb.databricks.com/libraries/replace-default-jar-new-jar.html) cluster directory"